extensions/wikia/VideoHandlers/apiwrappers/AnyclipApiWrapper.class.php

```diff
@@ -3,7 +3,7 @@
 class AnyclipApiWrapper extends ApiWrapper {
 
 	protected $ingestion = true;
-
+	// HTTPS-note: not switching to https, as apis.anyclip.com doesn't seem to work at all.
```

Comment:
Can we put these non-working/non-HTTPS ones in a tracking ticket to address/discuss/sunset and include the ticket in the comment, so we don't forget?

Reply:
Good point, I've created a follow up ticket